packages/dd-trace/src/plugins/util/web.js

```diff
@@ -202,9 +202,10 @@ function startSpan (tracer, config, req, res, name) {
 }
 
 function configureDatadogObject (tracer, span, req, res) {
-  req._datadog.tracer = tracer
-  req._datadog.span = span
-  req._datadog.res = res
+  const ddObj = req._datadog
```

Comment:
Did you really get a difference in performance here? v8 should be optimizing that already.

Reply:
This was an "every little bit helps" change. I can revert it if you want, but I'm pretty sure it doesn't hurt?